F21/rishith-cmykt/README

```diff
@@ -1 +1,23 @@
 Your program for the code review lab goes in this directory
+#ENGLSIH LANGUAGE
+y=input("Enter any word in uppercase letters : ")
+c=y.lower()
+print("Lower case letters : ",c)
+#IRISH LANGUAGE
+r = input("Enter any word in irish capital letters : ")
+if (r[0] == 'n' or r[0] == 't') and (r[1] == 'A' or r[1] == 'E' or r[1] == 'I' or r[1] == 'O' or r[1] == 'U' or r[1] == 'Á' or r[1] == 'É' or r[1] =='Í' or r[1] =='Ó' or r[1] == 'Ú'):
+    print(r[0].rstrip(),'-',r[1:].lower())
```

Comment:
by using commas instead of add the string together you are creating a space before and after your hyphen. try using r[0].rstrip()+'-'+r[1:].lower()

Reply:
Sure,I shall update the code